cmd/logger/logger.go

```diff
@@ -27,48 +27,42 @@ import (
 	"strings"
 	"time"
 
-	"github.com/fatih/color"
 	c "github.com/minio/mc/pkg/console"
+	"golang.org/x/crypto/ssh/terminal"
 )
 
-// global colors.
-var (
-	colorBold = color.New(color.Bold).SprintFunc()
-	colorRed  = color.New(color.FgRed).SprintfFunc()
-)
+const loggerTimeFormat string = "15:04:05 MST 01/02/2006"
 
 // Disable disables all logging, false by default. (used for "go test")
-var Disable = false
+var Disable = true
```

Comment:
Why true by default ?

Reply:
> Why true by default ?

I added this because sometimes the server prints some logs before printing the pretty error log and quit. Like in case of a corrupted FS format. That's why here I am hiding logs until the server completes the startup phase.